contents/tutorials/bounce-rate.md

```diff
@@ -1,102 +1,136 @@
 ---
 title: How to calculate bounce rate
-date: 2023-06-27
+date: 2024-12-11
 author:
   - ian-vanagas
+  - bijan-boustani
 showTitle: true
 sidebar: Docs
 tags:
   - insights
   - hogql
   - product analytics
+  - web analytics
 ---
 
-Bounce rate is the percentage of users who leave your page immediately after visiting. It is a popular marketing metric showing the relevance and engagement of content for site visitors.  
+Bounce rate is the percentage of users who leave a page immediately after visiting. It is a popular marketing metric showing the relevance and engagement of content for site visitors. This tutorial shows you how to calculate bounce rate in PostHog.
 
-This tutorial shows you how to calculate bounce rate in PostHog. To get started, you need to install the [snippet](/docs/getting-started/install?tab=snippet) or [JavaScript SDK](/docs/libraries/js) and enable "record user sessions"  in [project settings](https://app.posthog.com/project/settings).
+To get started, you need to install the [snippet](/docs/getting-started/install?tab=snippet) or [JavaScript SDK](/docs/libraries/js). And ensure the following settings are enabled in your project settings:
 
-> **How does Google Analytics 4 [calculate the bounce rate](https://support.google.com/analytics/answer/12195621?hl=en)?** It is the percentage of sessions that **do not** last longer than 10 seconds, have a conversion event, or have at least 2 pageviews or screenviews.
+- [Record user sessions](https://app.posthog.com/settings/environment#replay)
+- [Enable autocapture for web](https://app.posthog.com/settings/environment-autocapture#autocapture)
+
+## How do we define bounce rate?
+
+Your bounce rate is the percentage of sessions that resulted in a bounce. We define a bounce as a session where a visitor:
+
+- spent less than 10 seconds on a page
+- did not [autocapture](/docs/product-analytics/autocapture) any events
+- only had one pageview
+
+> **How does Google Analytics 4 [calculate bounce rate](https://support.google.com/analytics/answer/12195621?hl=en)?** It is the percentage of sessions that **do not** last longer than 10 seconds, have a conversion event, or have at least 2 pageviews or screenviews.
```

Comment:
Seems very similar to ours? Perhaps we should mention what the differences are

Reply:
Agreed. I rewrote this slightly to indicate that we're showing similarities instead of contrasting.
https://github.com/PostHog/posthog.com/pull/10106/files#diff-053c49ad5fdf99aeafb37ce839ecd20f3037aee42e833ed000974052a3e075aeR31